watson.completion

```diff
@@ -75,6 +75,18 @@ _watson_complete () {
             COMPREPLY=($(compgen -W "$tags" -- ${cur}))
           fi
           ;;
+        rename)
+          case "$3" in
+            -p|--project)
+              projects="$(watson projects)"
+              COMPREPLY=($(compgen -W "$projects" -- ${cur}))
```

Comment:
Can you update this to handle paces in project names properly, like in #122, please.